capabilities/version-control.md

```diff
@@ -1,28 +1,109 @@
+
+Automating infrastructure management ensures that infrastructure provisioning and configuration are reproducible and transparent as application code changes. This fosters better collaboration and consistency across development and operations teams.
 
 ### [Version Dependencies](/practices/version-dependencies.md)
-Version Dependencies complement the Version Control capability by ensuring dependencies are managed with the same diligence as source code. This practice enhances reproducibility, traceability, and consistency in software builds.
 
-### [Working in Small Increments](/practices/work-in-small-increments.md)
-Working in Small Increments improves the Version Control capability by facilitating more manageable, frequent, and understandable commits.
-This practice enhances the traceability and reversibility of changes, making it easier to identify and address issues quickly, thereby improving the overall efficiency and reliability of the development process.
-It also increases productivity and collaboration reducing the frequency of large merge conflicts.
\ No newline at end of file
+Version dependencies ensure dependencies are managed with the same diligence as source code. This practice enhances reproducibility, traceability, and consistency in software builds.
+
+### [Work in Small Increments](/practices/work-in-small-increments.md)
+
+Working in small increments enhances the traceability and reversibility of changes, making it easier to identify and address issues. This practice also increases productivity and collaboration, reducing the frequency of large merge conflicts. The result is a more efficient and reliable development process, as well as more manageable, frequent, and understandable commits.
+
+
+## Adjacent Capabilities
+
+The following capabilities will be valuable for you and your team to explore, as they are either:
+
+- Related (they cover similar territory to Version Control)
+- Upstream (they are a pre-requisite for Version Control)
+- Downstream (Version Control is a pre-requisite for them)
+
+### [Continuous Integration](/capabilities/continuous-integration.md) - Downstream
+
+Version Control is fundamental to Continuous Integration, as it provides the shared codebase where developers frequently integrate their changes. Effective use of version control allows teams to detect integration issues early and maintain a stable mainline, improving software quality and delivery speed.
+
+### [Trunk-Based Development](/capabilities/trunk-based-development.md) - Downstream
+
+Trunk-Based Development relies on a strong version control system to manage code changes in a single shared branch. By using version control effectively, teams can avoid complex merges and conflicts, facilitating rapid development and collaboration.
```

Comment:
Throughout the Capability, we refer to “version control” as both a system (explicitly) and as a practice. I sense the wording is correct in both cases; it's just situational. But would it be correct to standardize the wording to “version control” unless we’re talking about specific systems, like Git. 

For example, here, we would say: 
“Trunk-Based Development relies on strong version control...”
instead of
“Trunk-Based Development relies on a strong version control system...” 

Or am I splitting hairs?

Reply:
It's not a system. That's my bad. It is a category of software that helps people manage systems. Yeah, let's try to unify under the "version control" wording where possible.

Reply:
Copy that. Three instances of "version control systems" remain, as they seemed logical in the context.